fec.config.js

```diff
@@ -13,4 +16,22 @@ module.exports = {
    * Add additional webpack plugins
    */
   plugins: [],
-};
\ No newline at end of file
+  moduleFederation: {
+    exposes: {
+      './RootApp': path.resolve(__dirname, './src/AppEntry.tsx'),
+      './BookmarkedLearninResourcesWidget': path.resolve(
+        __dirname,
+        './src/components/LearningResourcesWidget/LearningResourcesWidget'
+      ),
+    },
+    exclude: ['react-router-dom'],
+    shared: [
+      {
+        'react-router-dom': {
+          singleton: true,
+          version: dependencies['react-router-dom'],
```

Comment:
OK, this should be a "*" then.

Reply:
@Hyperkid123 done; also fixed the typo in the name